tox.ini

```diff
@@ -14,7 +14,6 @@ tox_pyenv_fallback = false
 [pytest]
 filterwarnings =
     # Fail the tests if there are any warnings.
-    error
```

Comment:
I'm getting some warning from `pytest-xdist`. Need to remove this and fix the warning